contracts/oracle/src/errors.rs

```diff
@@ -2,7 +2,8 @@ use near_sdk::env::panic_str;
 use near_sdk::{base64, FunctionError};
 
 /// Contract errors
-#[cfg_attr(not(target_arch = "wasm32"), derive(Debug, PartialEq))]
+#[cfg_attr(not(target_arch = "wasm32"), derive(PartialEq))]
+#[derive(Debug)]
```

Comment:
```suggestion
#[cfg_attr(not(target_arch = "wasm32"), derive(PartialEq, Debug))]
```

Reply:
We need to derive the Debug because we are using `result.unwrap` in the `is_used_identity` method when we call `normalize_external_id`